src/x11/extensions/xkb.cpp

```diff
@@ -64,6 +64,18 @@ const string keyboard::layout_name(size_t index) const {
   return layouts[index].symbols[index];
 }
 
+/**
+ * Get current variant name
+ * "GROUP (VARIANT)"
+ *         ^^^^^^^
+ */
+const string keyboard::variant_name(size_t index) const {
+  string group_name = this->group_name(index);
+  int start = group_name.find('(') + 1;
+  int num_chars = group_name.find(')') - start;
+  return group_name.substr(start, num_chars);
```

Comment:
If there are no parentheses, then this just returns the entire group name. I think we should either return nothing or something like `N/A` in that case.